api/python/setup.py

```diff
@@ -1,13 +1,38 @@
+import os
+import sys
+
 from setuptools import setup, find_packages
+from setuptools.command.install import install
+
+VERSION = "0.0.3"
 
 def readme():
     readme_short = """
+    Quilt T4 is a data management tool designed for data discoverability, data dependency
+    management, and data version control using `data packages <https://blog.quiltdata.com/data-packages-for-fast-reproducible-python-analysis-c74b78015c7f>`_.
+
+    The `t4` PyPi package allows you to build, push, and pull data packages in T4 using Python.
+    Visit the GitHub `repository <https://github.com/quiltdata/t4>`_ for more information.
```

Comment:
Now that docs are up we can update this link text to:

```
Visit the `documentation quickstart <https://quiltdocs.gitbook.io/t4/quickstart>`_ for more information.
```

Reply:
Done, thanks